Backends/PDF/paper.lisp

```diff
@@ -35,13 +35,22 @@
       (rotatef width height))
     (make-rectangle* 0 0 width height)))
 
+(defparameter *pdf-top-margin* 1)
+(defparameter *pdf-left-margin* 1)
+(defparameter *pdf-bottom-margin* 1)
+(defparameter *pdf-right-margin* 1)
+
```

Comment:
Yes, the margin size should be customizable. I'm not sure I want to commit to a particularly exported interface yet though. I much prefer the special var's to, e.g., (- height 2) and a comment. I'd like to see this committed as is and some subsequent thought given to how spec margins for printed objects across both PS and PDF backends.